### ISSUE TITLE
치지직 마지막 방송일때5초간격 영상 짤리는 현상이 나옵니다 

### ISSUE THREAD

**Issue by subakir132** [KurtBestor/Hitomi-Downloader#7015]
![화면 캡처 2024-03-14 184932](https://github.com/KurtBestor/Hitomi-Downloader/assets/162278689/c9b07d4c-3fc2-457c-9c26-c8138c194375)
[11.txt](https://github.com/KurtBestor/Hitomi-Downloader/files/14600433/11.txt)

녹화 잡을려고 했을 때

![화면 캡처 2024-03-14 190146](https://github.com/KurtBestor/Hitomi-Downloader/assets/162278689/0c8fbb0d-5bea-45cd-a35a-88d18257036b)

[211.txt](https://github.com/KurtBestor/Hitomi-Downloader/files/14600548/211.txt)

**Comment by KurtBestor** [comment]
첫번째 증상은 Chzzk에서 방송이 끝나고도 30초 ~ 1분 정도 방송중이라고 응답해서 생기는 문제로 정상이에요
VPN 이나 프록시 같은 것을 쓰나요?

**Comment by subakir132** [comment]
> 첫 번째 특징은 Chzzk에서 방송이 벨소리도 30초 ~ 1분 정도 방송중이라고 응답하는 문제와 같은 것입니다 VPN 이나 그것을 쓰나요?


아니요, 10초 단위로 녹화하게 끔 설정했고, VPN, 프록시는 쓰지 않았습니다. 근데 유독 치지직에서 해당 방송 막바지 때 5초씩 녹화하는 현상이 생깁니다
